controller-client/src/paginator.rs

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+use crate::{ControllerClient, ControllerError, ResultRetry};
+use futures::prelude::*;
+use futures::stream::{self};
+use pravega_client_retry::retry_result::RetryError;
+use pravega_client_shared::Scope;
+use std::vec::IntoIter;
+use tracing::info;
+
+///
+///Helper method to iterated over the all the Pravega streams under the provided Scope.
+///This method returns a stream of values,Pravega streams, produced asynchronously.
+///
+/// The below snippets show case the example uses.
+/// Sample 1:
+///```ignore
```

Comment:
Do you want `ignore` or `no_run`?

Reply:
i used `ignore` since

Reply:
Oh, yeah that a different crate. Sorry I don't think there is a way around that which doesn't involve adding a dev-dependency.